sg.tf

```
@@ -53,7 +53,7 @@ locals {
 
 module "security_group" {
   source  = "cloudposse/security-group/aws"
-  version = "1.0.1"
+  version = "2.0.0"
```

This module should wait until https://github.com/cloudposse/terraform-aws-security-group/issues/43 is fixed and then use the fixed module.

```suggestion
  version = "2.0.1"
```

---

Any ETAs on when that will be completed?